### ISSUE TITLE
AOL extractor failing

### ISSUE THREAD

**Issue by SeanPM5** [rg3/youtube-dl#9451]
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2016.05.10**
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)

```
iMac:Desktop Sean$ youtube-dl -v http://on.aol.com/partners/abc-551438d309eab105804dbfe8/eddie-gets-mad-at-fun-mom-jessica-5732b390e4b06a26f876cd92
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-v', u'http://on.aol.com/partners/abc-551438d309eab105804dbfe8/eddie-gets-mad-at-fun-mom-jessica-5732b390e4b06a26f876cd92']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2016.05.10
[debug] Python version 2.7.11 - Darwin-15.3.0-x86_64-i386-64bit
[debug] exe versions: avconv 11.4, avprobe 11.4, ffmpeg 3.0.2, ffprobe 3.0.2, rtmpdump 2.4
[debug] Proxy map: {}
[on.aol.com] 5732b390e4b06a26f876cd92: Downloading JSON metadata
[on.aol.com] 5732b390e4b06a26f876cd92: Downloading m3u8 information
[on.aol.com] 5732b390e4b06a26f876cd92: Downloading m3u8 information
[debug] Invoking downloader on u'http://content-ause2.uplynk.com/e61d5a73e47c4cea84311ae946214d04/h.m3u8?hdnea=exp=1463027929~acl=/*~hmac=de62bccd97525e1090af38c06c654db2ddf359e6aabe555ac6f72375cff9c9bc'
[download] Destination: Eddie Gets Mad at 'Fun Mom' Jessica-5732b390e4b06a26f876cd92.mp4
[debug] avconv command line: avconv -y -headers 'Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7
Accept-Language: en-us,en;q=0.5
Accept-Encoding: gzip, deflate
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:10.0) Gecko/20150101 Firefox/44.0 (Chrome)
' -i 'http://content-ause2.uplynk.com/e61d5a73e47c4cea84311ae946214d04/h.m3u8?hdnea=exp=1463027929~acl=/*~hmac=de62bccd97525e1090af38c06c654db2ddf359e6aabe555ac6f72375cff9c9bc' -c copy -f mp4 -bsf:a aac_adtstoasc 'file:Eddie Gets Mad at '"'"'Fun Mom'"'"' Jessica-5732b390e4b06a26f876cd92.mp4.part'
avconv version 11.4, Copyright (c) 2000-2014 the Libav developers
  built on Mar  1 2016 12:20:19 with Apple LLVM version 7.0.2 (clang-700.1.81)
Unable to open key file https://content-ause2.uplynk.com/check2?b=e61d5a73e47c4cea84311ae946214d04&v=e61d5a73e47c4cea84311ae946214d04&r=h
Format detected only with low score of 1, misdetection possible!
[mp3 @ 0x7ff5ea838c00] Header missing
    Last message repeated 1067 times
[mp3 @ 0x7ff5ea830600] Estimating duration from bitrate, this may be inaccurate
[mp2 @ 0x7ff5ea9dd800] Header missing
    Last message repeated 1067 times
Input #0, hls,applehttp, from 'http://content-ause2.uplynk.com/e61d5a73e47c4cea84311ae946214d04/h.m3u8?hdnea=exp=1463027929~acl=/*~hmac=de62bccd97525e1090af38c06c654db2ddf359e6aabe555ac6f72375cff9c9bc':
  Duration: 00:01:10.50, bitrate: 0 kb/s
  Program 0 
    Metadata:
      variant_bitrate : 0
    Stream #0.0: Audio: mp2, 0 channels, s16p
Output #0, mp4, to 'file:Eddie Gets Mad at 'Fun Mom' Jessica-5732b390e4b06a26f876cd92.mp4.part':
Output file #0 does not contain any stream


ERROR: ffmpeg exited with code 1
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/bin/youtube-dl/__main__.py", line 19, in <module>
    youtube_dl.main()
  File "/usr/local/bin/youtube-dl/youtube_dl/__init__.py", line 421, in main
    _real_main(argv)
  File "/usr/local/bin/youtube-dl/youtube_dl/__init__.py", line 411, in _real_main
    retcode = ydl.download(all_urls)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1735, in download
    url, force_generic_extractor=self.params.get('force_generic_extractor', False))
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 687, in extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 732, in process_ie_result
    return self.process_video_result(ie_result, download=download)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1381, in process_video_result
    self.process_info(new_info)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1643, in process_info
    success = dl(filename, info_dict)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1585, in dl
    return fd.download(name, info)
  File "/usr/local/bin/youtube-dl/youtube_dl/downloader/common.py", line 350, in download
    return self.real_download(filename, info_dict)
  File "/usr/local/bin/youtube-dl/youtube_dl/downloader/external.py", line 43, in real_download
    self.get_basename(), retval))
  File "/usr/local/bin/youtube-dl/youtube_dl/downloader/common.py", line 161, in report_error
    self.ydl.report_error(*args, **kargs)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 550, in report_error
    self.trouble(error_message, tb)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 512, in trouble
    tb_data = traceback.format_list(traceback.extract_stack())

...
<end of log>
```

### Description of your *issue*, suggested solution and other information

AOL On website is no longer downloading videos as of today and needs to be updated. It seems to extract the correct links still, but as of today says "Unable to open key file" and then fails.

Example link to test: http://on.aol.com/video/steph-curry-is-nba-s-1st-unanimous-mvp-57322966e4b0144970cadead

**Comment by yan12125** [comment]
Seems some videos are encrypted. Beofre it's fixed, use the following command to download on.aol.com videos:
```
youtube-dl -v -f 'best[protocol^=http]' "http://on.aol.com/video/steph-curry-is-nba-s-1st-unanimous-mvp-57322966e4b0144970cadead"
```

**Comment by SeanPM5** [comment]
Thanks a lot @yan12125 that special command works perfectly for now until it's fixed.

**Comment by rhnvrm** [comment]
`youtube-dl http://on.aol.com/video/steph-curry-is-nba-s-1st-unanimous-mvp-57322966e4b0144970cadead` works for me now. 

It has probably been fixed see #9551 by https://github.com/rg3/youtube-dl/commit/0b36a96212861b3fc3f11b26c22216ed3cc4d2ce

We should consider closing this issue.

**Comment by yan12125** [comment]
It's working now as people at on.aol.com has fixed their website. Closing.
